controllers/designateapi_controller.go

```diff
@@ -357,13 +357,13 @@ func (r *DesignateAPIReconciler) reconcileInit(
 	// expose the service (create service, route and return the created endpoint URLs)
 	//
 	var designatePorts = map[endpoint.Endpoint]endpoint.Data{
-		endpoint.EndpointAdmin: endpoint.Data{
+		endpoint.EndpointAdmin: {
 			Port: designate.DesignateAdminPort,
 		},
```

Comment:
for all other services we stopped creating the admin endpt, e.g. https://github.com/openstack-k8s-operators/placement-operator/blob/master/controllers/placementapi_controller.go#L328-L334 . @weinimo ok to not have the admin endpt for designate?

Reply:
@dkehn what do you think?
(Sorry, I am not very familiar with Designate's design)